ui/common/css/header/_header.scss

```diff
@@ -38,11 +82,12 @@
   &__donate {
     @extend %flex-center;
     color: $c-brag;
-    margin-left: 1rem;
+    margin: 0 1rem;
     text-transform: uppercase;
     line-height: $site-header-height;
-    @include breakpoint($mq-not-xx-small) {
-      display: none;
+
+    @include breakpoint($mq-not-x-small) {
+      margin-#{$start-direction}: calc($site-header-height + 4px);
```

Comment:
horizontal margin usually depends on page width, not height. Like `1vw`.
In any case I don't think we should displaying this link at all here.

Reply:
That's your code dude.